db/seeds.d/07-provisioning_templates.rb

```diff
@@ -95,6 +95,8 @@
     contents = File.read(File.join("#{Rails.root}/app/views/unattended", input.delete(:source)))
 
     if (t = ProvisioningTemplate.unscoped.find_by_name(input[:name])) && !audit_modified?(ProvisioningTemplate, input[:name])
+      next if t.global_default?
```

Comment:
@xprazak2 why was this skipping added? if the recommended workflow now is to clone the template and change the link it via settings why do we skip this? if user is fine with the default template (which would be locked), it would not be updated it with newer version I think